sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -139,6 +139,28 @@ DOCS=( CONTRIBUTING.md README.md )
 
 CHECKREQS_DISK_BUILD="3G"
 
+MY_CUDA_ARCH=
+
+get_cuda_arch() {
```

Comment:
I assume you're using a function with the idea that the cuda eclass will replace it eventually, but global scope functions should bit more private namespacing like vtk_ or my_

Also, functions used only in a single phase are normally defined inside the phase itself to have less in global scope. There's even been talk to make this a pkgcheck warning.

...that aside, personally feel this shouldn't be a function if called only once, not like it'll be hard to delete the block and replace by a cuda.eclass function call if it ever exists.

Reply:
For one, I'm using a function with the idea of extending the cuda eclass.
Second, it makes it more modular. I like my ebuilds clean and do not like the idea of having a if-clause containing a switch clause, all within an optional `if use cuda; then` statement, deeply nested inside a phase function.